source/sessions/tests/dirty-session-errors.yml

```diff
@@ -11,96 +11,6 @@ data:
   - {_id: 1}
 
 tests:
-  - description: Clean explicit session is not discarded
```

Comment:
Two tests are still being removed here:

 * Clean explicit session is not discarded
 * Clean implicit session is not discarded

Is that intentional?

Reply:
Yes it was intentional, they were actually just renamed to be the tests in the `server-support` tests